entries/checked-selector.xml

```diff
@@ -70,6 +71,40 @@ $( "input" ).on( "click", function() {
 </form>
 ]]></html>
   </example>
+  <example>
+    <desc>Determine how many input elements are checked.</desc>
+    <code><![CDATA[
+var countCheckedOptions = function() {
+  var n = $( "option:checked" ).length;
+  $( "#log_option" ).text( n + (n === 1 ? " is" : " are") + " ':checked'!" );
+};
+countCheckedOptions();
+
+$( "select" ).on( "change", countCheckedOptions );
+]]></code>
+    <css><![CDATA[
+  #log_option {
+    color: red;
+  }
+]]></css>
+    <html><![CDATA[
+<form>
+  <label for="cities">Cities I'd like to visit</label>
+  <br />
+  <select name="cities" id="cities" size="8" multiple="multiple">
+    <option value="Athens">Athens</option>
+    <option value="Barcelona">Barcelona</option>
+    <option value="Chicago">Chicago</option>
+    <option value="Detroit">Detroit</option>
+    <option value="Frankfurt">Frankfurt</option>
+    <option value="London">London</option>
+    <option value="Madrid">Madrid</option>
+    <option value="Paris">Paris</option>
+  </select>
+</form>
+<div id="log_option"></div>
+]]></html>
+  </example>
```

Comment:
@arthurvr How do you you mean? The example demonstrates how the `:checked` selector can be used with `option` elements; this demonstration is missing from the current documentation.

Do you want me to separate into two tickets: #558 and a new ticket for giving an example of using `:checked` with an `option` element?